test/Extensions/TesterAzureUtils/Persistence/PersistenceProviderTests.cs

```diff
@@ -202,7 +202,7 @@ public async Task AzureTableStorage_ConvertToFromStorageFormat(int? stringLength
             Assert.Equal(initialState.C, convertedState.C);
         }
 
-        [Fact, TestCategory("Functional"), TestCategory("Azure")]
+        [SkippableFact, TestCategory("Functional"), TestCategory("Azure")]
```

Comment:
what's the skipping condition?  Mark it as `SkippableFact` wouldn't necessary skip it.